src/test/java/ibm/jceplus/junit/base/memstress/BaseTestMemStressRSAPSS2.java

```diff
@@ -89,7 +84,7 @@ public void testRSAPSSSignature() throws Exception {
         long currentUsedMemory = 0;
         long prevUsedMemory = 0;
 
-        KeyPairGenerator keyGen = KeyPairGenerator.getInstance("RSA", providerName);
+        KeyPairGenerator keyGen = KeyPairGenerator.getInstance("RSA", getProviderName());
         keyGen.initialize(this.keysize, new java.security.SecureRandom());
```

Comment:
Shouldn't this change to `getKeySize()` too?

Reply:
Yes agreed. Change this test to use getKeySize() instead such that this is more consistent.